### PR TITLE
v1.0.1: Log Braze impressions on presenting Judo-powered IAM campaigns

### PR DESCRIPTION
**Pull request by orospakr**
As per the Braze [documentation for custom In-App Messaging UIs](https://www.braze.com/docs/developer_guide/platform_integration_guides/ios/in-app_messaging/customization/#logging-impressions-and-clicks) (that replace the stock Braze one), such custom UIs are responsible for calling Braze's impression log routine.

This was neglected in the initial implementation of the module.

Note: since this is a hotfix release, and the way we release Android libraries is to use GitHub Pages to host a flat file Maven repo via HTTP, this PR also includes the version number bump and a build of the new artifacts.

### CODE REVIEW

File: sample/src/main/java/app/judo/braze/sample/JudoBrazeSampleApplication.kt [hunk 1]
```diff
@@ -17,7 +17,7 @@ class JudoBrazeSampleApplication: Application() {
         Judo.initialize(
             application = this,
             accessToken = "<JUDO ACCESS TOKEN>",
-            "<JUDO APP DOMAIN>"
```
Comment (Moux) [inline sample/src/main/java/app/judo/braze/sample/JudoBrazeSampleApplication.kt]: This should be reverted, the judo app domain has been left in.

File: sample/src/main/res/values/braze.xml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <string name="com_appboy_api_key">YOUR_APP_IDENTIFIER_API_KEY</string>
-    <string translatable="false" name="com_appboy_custom_endpoint">YOUR_CUSTOM_ENDPOINT_OR_CLUSTER</string>
+    <string name="com_appboy_api_key">ae51827d-ee15-46d1-80c4-6b8b0593a409</string>
```
Comment (Moux) [inline sample/src/main/res/values/braze.xml]: These changes should be reverted, as your keys and endpoint are here.

File: build-release.sh [hunk 3]
```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+JAVA_HOME=/Applications/Android\ Studio.app/Contents/jre/Contents/Home ANDROID_HOME=~/Library/Android/sdk ./gradlew clean publishReleasePublicationToMavenRepository
```
Comment (Moux) [inline build-release.sh:3]: Should this file be part of the release?

**Review by Moux — CHANGES_REQUESTED** [review]

File: sample/src/main/res/values/braze.xml (hunk 2 above)
Comment (orospakr) [reply sample/src/main/res/values/braze.xml]: dangit thanks. Thought I avoided that. Famous last words, lol.

File: build-release.sh (hunk 3 above)
Comment (orospakr) [reply build-release.sh:3]: ya I realized I hadn't documented the build anywhere in this particular repo

**Review by Moux — APPROVED** [review]

**Pull request merged**